src/QuixStreams.Streaming/QuixStreamingClient.cs

```diff
@@ -41,8 +43,10 @@ public interface IQuixStreamingClient : IQuixStreamingClientAsync
         /// <param name="consumerGroup">The consumer group id to use for consuming messages. If null, consumer group is not used and only consuming new messages.</param>
         /// <param name="options">The settings to use for committing</param>
         /// <param name="autoOffset">The offset to use when there is no saved offset for the consumer group.</param>
+        /// <param name="partition">The partition to read from</param>
+        /// <param name="offset">The offset to start reading from</param>
         /// <returns>Instance of <see cref="ITopicConsumer"/></returns>
-        ITopicConsumer GetTopicConsumer(string topicIdOrName, string consumerGroup = null, CommitOptions options = null, AutoOffsetReset autoOffset = AutoOffsetReset.Latest);
+        ITopicConsumer GetTopicConsumer(string topicIdOrName, string consumerGroup = null, CommitOptions options = null, AutoOffsetReset autoOffset = AutoOffsetReset.Latest, Partition? partition = null, Offset? offset = null);
```

Comment:
auto offset reset and offset at the same time doesn't make sense.
Auto offset reset is just setting offset via enum (if consumer doesn't have it already)

Same for the other endpoints